project-template/cookiecutter.json

```diff
@@ -5,8 +5,8 @@
   "pypi_package_name": "exasol-{{cookiecutter.repo_name}}",
   "import_package": "exasol.{{cookiecutter.package_name}}",
   "description": "",
-  "author_full_name": "",
-  "author_email": "",
+  "author_full_name": "{{cookiecutter.author_full_name}}",
```

Comment:
Needed as the new test `test_poetry_check_passes` finds that without these properly filled that our template breaks.